README.md

```diff
@@ -0,0 +1,139 @@
+<a name="readme-top"></a>
```

Comment:
![image](https://github.com/haftamudesta/Blog-App/assets/93039359/03caef22-6da9-43bd-ae04-13c0f310041f)

- [ ] It would be nice if add the details of the work which you have done in this milestone in the pull request summary instead of adding the description of the project to make your pull request professional 👍